training-export-service/test_export.py

```
@@ -2,10 +2,58 @@
+
+    def _z(self, x, y, w, h):
+        return {'bounds': {'x': x, 'y': y, 'w': w, 'h': h}}
+
+    def test_whole_plus_cells_merge_to_whole_box(self):
+        whole = self._z(70, 98, 470, 338)                              # Kim p191 style
+        cells = [self._z(250 + i * 8, 120, 6, 8) for i in range(20)]   # cells inside
+        merged = merge_table_cells_into_tables([whole] + cells)
+        self.assertEqual(len(merged), 1)
+        b = merged[0]['bounds']
+        # Union equals the whole-table box (the cells sit inside it).
+        self.assertAlmostEqual(b['x'], 70);  self.assertAlmostEqual(b['y'], 98)
```

Ruff (0.15.17)

[error] Multiple statements on one line (semicolon)

(E702)